<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>usersearch</class>
 <widget class="QWidget" name="usersearch">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>641</width>
    <height>543</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>601</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <family>宋体</family>
       </font>
      </property>
      <property name="text">
       <string>身份证号：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_id"/>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_search">
      <property name="font">
       <font>
        <family>Agency FB</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>查询</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_exit">
      <property name="font">
       <font>
        <family>Agency FB</family>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>返回</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
    </font>
   </property>
   <property name="text">
    <string>户口信息：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>300</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>户籍成员信息：</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>601</width>
     <height>81</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableView" name="tableView_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>320</y>
     <width>601</width>
     <height>201</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableView" name="person">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>601</width>
     <height>81</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
    </font>
   </property>
   <property name="text">
    <string>个人信息：</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineEdit_id</tabstop>
  <tabstop>pushButton_search</tabstop>
  <tabstop>pushButton_exit</tabstop>
  <tabstop>person</tabstop>
  <tabstop>tableView</tabstop>
  <tabstop>tableView_2</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
